### PR TITLE
Support reading single draft by id

### PR DESCRIPTION
**Pull request by rkondratowicz**

### CODE REVIEW

File: src/test/draft/draftStoreClient.ts [hunk 1]
```diff
@@ -70,6 +71,34 @@ describe('DraftStoreClient', () => {
...
+        draftStoreServiceMock.resolveReadOne(modelFromApi)
+
+        const client: DraftStoreClient<any> = await factory.create(draftStoreServiceMock.serviceBaseURL, request)
+        const draft: Draft<any> = await client.readOne('123', 'token', (doc => doc))
+
+        expect(draft.id).to.equal(modelFromApi.id)
+        expect(draft.type).to.equal(modelFromApi.type)
+        expect(draft.document).to.deep.equal(modelFromApi.document)
+        expect(draft.created).to.deep.equal(moment(modelFromApi.created))
+        expect(draft.updated).to.deep.equal(moment(modelFromApi.updated))
```
Comment (rkondratowicz) [inline src/test/draft/draftStoreClient.ts]: does mocha have something similar to assertj's soft assertions?

File: src/main/draft/draftService.ts [hunk 2]
```diff
@@ -35,6 +35,18 @@ export class DraftService {
     return client.find({ type: draftType, limit: limit }, userToken, deserializationFn, secrets)
   }
 
+  async readOne<T> (
```
Comment (timja) [inline src/main/draft/draftService.ts]: I think this should just be called `read`

Comment (rkondratowicz) [reply src/main/draft/draftService.ts]: done

File: src/test/http-mocks/draft-store.ts [hunk 3]
```diff
@@ -32,6 +32,12 @@ export function resolveFindNoDraftFound (): mock.Scope {
     })
 }
 
+export function resolveReadOne (body: any): mock.Scope {
```
Comment (timja) [inline src/test/http-mocks/draft-store.ts]: resolveRead?

**Review by timja — APPROVED** [review]

File: src/test/http-mocks/draft-store.ts (hunk 3 above)
Comment (rkondratowicz) [reply src/test/http-mocks/draft-store.ts]: 👍

**Pull request merged**